tests/apitest/src/AppKit/NSToolbarItem.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using NUnit.Framework;
+
+#if !XAMCORE_2_0
+using MonoMac.AppKit;
+using MonoMac.ObjCRuntime;
+using MonoMac.Foundation;
+#else
+using AppKit;
+using ObjCRuntime;
+using Foundation;
+#endif
+
+namespace Xamarin.Mac.Tests
+{
+	public class NSToolbarItemTests
+	{
+		[Test]
+		public void InitTests ()
+		{
+			NSSToolbarItem item = new NSToolbarItem ();
+			Assert.IsNotNull (item);
```

Comment:
It might throw inside `new` but C# won't ever return `null` for a `new`. Instead double check that Handle is not null and that all properties can be retrieved (i.e. that this is not an invalid, or half initialized native instance).

Also add a unique string message to each `Assert`, so we know which failed even if debug symbols are unavailable (or broken).